### PR TITLE
add Howie66 as an assignee into this task

### PR DESCRIPTION
**Pull request by Howie66**

### CODE REVIEW

**Comment by Howie66** [comment]
> /assign @feynmanzhou

/assign @feynmanzhou

File: sig-docs/installation/community-collaboration-for-v3.0.0.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ If you would like to contribute to community, raise a PR to add yourself as an a
 
 | Task ID | Task | Assignee (GitHub ID) | Target Date | Preferred Language | PR link |
 | --- | --- | --- | --- | --- | --- |
-| # 1 | KubeSphere on Aliyun ECS | TBD  | TBD | TBD | TBD |
+| # 1 | KubeSphere on Aliyun ECS | Howie66  | Aug.17| English | https://github.com/Howie66/community |
```
Comment (FeynmanZhou) [inline sig-docs/installation/community-collaboration-for-v3.0.0.md:11]: Please update your GitHub ID with its link: [Howie66](https://github.com/Howie66/), and remove the PR link. The `PR link` means the final documentation of your Pull Request, you can fill it out when you complete it.

**Comment by FeynmanZhou** [comment]
Please add a signoff within your git commit, take an example:

```
git commit -s -m "add my commit information"
```

**Comment by FeynmanZhou** [comment]
Still lost the sign-off.

**Comment by Howie66** [comment]
Signed-off-by: Howie66 haoweili35@gmail.com

**Pull request closed without merging**